locale-theme/app.pot

```diff
@@ -88,7 +88,7 @@ msgid "If {{email_address}} is the wrong address for information requests to {{p
 msgstr ""
 
 msgid "Internal Review"
-msgstr ""
+msgstr "Confirmatory Application"
```

Comment:
In general its better to edit in transifex and then [pull new changes](https://github.com/mysociety/alaveteli/wiki/Update-theme-translations) in to ensure you don't overwrite translations.